[ALMOST READY] Blueshift Gauntlets 2: Electric Boogaloo

gonenoculer5 opened the pull request:
## About The Pull Request
Introducing a new method of surgery: _**The Blueshift Gauntlets!**_

These mythical bluespace tech gloves allow for a surgeon (or paramedic!) to preform effective surgeries on the fly or in the field, allowing for emergency surgery on a table or other surface in the case of illnesses such as terminal appendicitis, heart attacks, or just the casual cavity implant.

**_For the moment, this is all it will be capable of, unless I end up adding more functions to it before this releases._**

## Why It's Good For The Game

Personally, I have always thought it would be interesting to see paramedics have more tools to use in their arsenal that would facilitate less ghetto field surgery.

## Credits:

**Code by:**
Gonenoculer5
**Spriting done by:**
Papaporo Paprito

## Changelog
:cl:
add: The Blueshift Gauntlets, a pair of hyper-advanced gloves capable of manipulating bluespace folds to speed up and enhance surgery in the field.
tweak: Modified many surgery dm's to permit the usage of the BSG's as a surgery tool.
/:cl:

gonenoculer5 committed:
Merge pull request #4 from TheSwain/master

upd8

gonenoculer5 committed:
Merge pull request #5 from TheSwain/master

update part 2 million hyes

gonenoculer5 committed:
Merge pull request #6 from TheSwain/master

update

gonenoculer5 commented:
proper version of pull request https://github.com/TheSwain/Fulpstation/pull/384

gonenoculer5 committed:
BSG Groundwork

Lays out the basic groundwork for the item, martial art, and formatting for future inclusions.

gonenoculer5 committed:
Update bsg.dm

Forgot to remove some //'s

gonenoculer5 committed:
Gauntlet Refactor

Refactors the gauntlets to be cleaner, removes the now outdated help document since its less complicated to add new moves, modifies existing surgery steps to accept the Phantom Manipulators, Holographic Drapes, and Holographic Cautery.

ATH1909 commented on code/FulpstationCode/blueshift_gauntlets/bsg.dm:
```diff
@@ -0,0 +1,80 @@
+		owner.visible_message("[owner.name] tries to summon a tool while the gauntlets are off with a dull beep.")
+		playsound(owner,'sound/machines/buzz-two.ogg',1,TRUE)
+	else
+		to_chat(owner, "<span class ='notice'>You deactivate the gauntlets, removing them from the bluespace fold!</span>")
```
?

gonenoculer5 replied:
the gauntlets turn themselves off if you take them off while they're on is what that does essentially, prevents the var getting stuck turned on after being removed or other werid scenarios like that by resetting the var

gonenoculer5 committed:
a

ATH1909 commented on code/FulpstationCode/blueshift_gauntlets/bsg.dm:
```diff
@@ -0,0 +1,105 @@
+	if(isPhased == FALSE)
+		to_chat(owner, "<span class ='notice'>You activate the gauntlets, preparing them to phase!</span>")
+		isPhased = TRUE
+		owner.visible_message("[owner.name] activates their Blueshift Gauntlets with a low hum.")
```
you could use the sound effect for turning a stasis bed on (NOT the sound effect for BUCKLING someone to a stasis bed) here for the humming noise

gonenoculer5 replied:
Yeah! I was considering using that, there are a few other candidates but that one i like.

gonenoculer5 committed:
a

gonenoculer5 committed:
a

gonenoculer5 committed:
Update bsg.dm

gonenoculer5 committed:
Update bsg.dm

gonenoculer5 committed:
Implement list

a

gonenoculer5 committed:
Finalization

Adds support for emagging for the next PR on this, and removes BSG_SURGERIES.DM, splits the action buttons into two seperate buttons for cleaner presentation and a less messy solution, and finally adds the sound effects for the activation.

gonenoculer5 committed:
Cleanup Pt1

Go around with final testing and clean up alot of grammar, and finally fix that nagging error with the BSG's being able to spawn two pairs of manipulators.

gonenoculer5 commented:
@remove32 I dont know how to resolve merge conflicts, mind lending me a hand here?

gonenoculer5 commented:
Currently waiting on sprites from Papaporo to release this, but other then that its done!

gonenoculer5 committed:
Merge branch 'master' into Blueshift2

gonenoculer5 commented:
Fixed merge conflicts

gonenoculer5 commented:
Waited several weeks for sprites, no sprites, going to just migrate this to my personal CB.
